src/pages/debug-app/debug-app.tsx

```diff
@@ -5,45 +5,39 @@ import {
   TileWarning,
```

Comment:
Point to layer logic is not working for selected tilesets in Viewer and in Debug.
We need to use `findExampleAndUpdateWithViewState` function. 
I guess we can move this function to the utils and use everythere.